backend/app/schemas/role.py

```diff
@@ -9,6 +9,7 @@ class RoleBase(BaseModel):
     role_name: str
 
 
+
```

Comment:
/nit Is this intended? Does pre-commit remove this @jonathantan1425 ?

Reply:
I think it should? I'm not very sure